strings/word_occurence2.py

```diff
@@ -8,8 +8,8 @@
 def word_occurence(sentence: str) -> dict:
     """
     >>> from collections import Counter
-    >>> all(word_occurence(s) == Counter(s.split()) for s in
-    ...     ("a b a b c a", "I love to love Python ❤️ 💕 ❤️", ""))
+    >>> all(word_occurence(s) == Counter(s.split(' ')) for s in
```

Comment:
What happens when the input string has two spaces in a row? `"a a  a b  b"`  --  I think we are better off using __s.split()__ than __s.split(" ")__

Reply:
I think line 17 `unique_words = list(set(words))` helps us to solve this issue.